### PR TITLE
fix(business/checkbox): fill block of indeterminate

### PR DESCRIPTION
**Pull request by efux**
This shows the indeterminate state of the business checkbox as block with background color. If someone tries to resize the checkbox, this is necessary to prevent a wrong look. This is precautionary.

### CODE REVIEW

File: projects/sbb-esta/angular-public/src/lib/checkbox/_checkbox.scss [hunk 1]
```diff
@@ -65,6 +65,7 @@ $checkBoxContainerBorder: 2;
           left: 4px;
           top: 8.5px;
           border: 1px solid $sbbColorGranite;
+          background-color: $sbbColorGranite;
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/src/lib/checkbox/_checkbox.scss]: Why was this necessary?

Comment (efux) [reply projects/sbb-esta/angular-public/src/lib/checkbox/_checkbox.scss]: This might be related to #141 . On my screen the checkbox was bigger then it should be and the indeterminate state was displayed only as a grey border instead of a filled grey box. This might be  redundant, when #141 is fixed and the checkbox always has the pre-defined size.

**Comment by efux** [comment]
I thought this might be related to #141, however #231 did not fix it so I investigated further and found the issue. It was not related to this library at all, instead somewhere `box-sizing: border-box` was overwritten.

Therefore I close this PR.
